pkg/util/k8sutil/backup.go

```diff
@@ -36,7 +36,9 @@ import (
 )
 
 const (
-	storageClassPrefix        = "etcd-backup"
+	// StorageClassPrefix is the prefix used when creating custom storage classes
+	// used in backup operations
```

Comment:
StorageClassPrefix is the prefix used when creating custom storage classes for backups through a given provisioner.